### DISCUSSION TITLE
🖥 | Admin Panel UI

### DISCUSSION THREAD

**Discussion by EhsanFox** [SeTar-Bot/Setar-Bot#73] [Q&A] [answered]
### Admin UI Panel 🖥 

is there any way we can make our UI of Admin Panel something similar to Discord itself? 
Such as [Discords City](https://discords.city)

Also, don't forget to [Join our Discord Community](https://discord.gg/7jgfP6j4Tc)

**Comment by im-parsa** [comment, marked as answer]
Hey @EhsanFox 👋🏻 ,

Yes, [Discords City](https://discords.city)'s developers now working on it.

You can use it on React.js/Next.js, Vue.js, and Angular.js but now it is not released
